cli/dcoscli/package/main.py

```diff
@@ -401,7 +401,14 @@ def _list(json_, endpoints, app_id, package_name):
 
             results.append(pkg_info)
 
-    emitting.publish_table(emitter, results, tables.package_table, json_)
+    if results:
+        emitting.publish_table(emitter, results, tables.package_table, json_)
+    else:
+        msg = ("There are currently no installed packages. "
+               "Please use `dcos package install` to install a package.")
+        if json_:
+            msg = json.dumps(msg)
```

Comment:
I don't think this is going to work. In the "success" case it will return an JSON array. In the "failure" case it will return a JSON string. Let's try to keep this stuff consistent.